packages/ui-components/src/multiselect/multiselect.tsx

```diff
@@ -375,7 +400,6 @@ function MultiSelect({ maxWidth = '100%', maxRows = 3, ...props }: MultiSelectPr
                         <Input
                             {...getInputProps(getDropdownProps({ preventKeyAction: isOpen }))}
                             disabled={props.disabled}
-                            onFocus={openMenu}
```

Comment:
Is this not required anymore?

Reply:
When I press Tab it automatically opens the menu, yes. Is there something else this was accounting for?

Reply:
No I think that's it, looks good then.